Gems/AWSCore/Code/Source/Editor/UI/AWSCoreResourceMappingToolAction.cpp

```diff
@@ -11,124 +11,141 @@
+        AzFramework::StringFunc::Path::Normalize(m_toolQtBinDirectoryPath);
+        if (!AZ::IO::SystemFile::Exists(m_toolQtBinDirectoryPath.c_str()))
+        {
+            AZ_Error(
+                AWSCoreResourceMappingToolActionName, false, "Failed to find ResourceMappingTool Qt binaries at %s.",
+                m_toolQtBinDirectoryPath.c_str());
+            m_toolQtBinDirectoryPath.clear();
         }
     }
 
     AZStd::string AWSCoreResourceMappingToolAction::GetToolLaunchCommand() const
     {
-        if (m_enginePythonEntryPath.empty() || m_toolScriptPath.empty() || m_toolQtBinDirectoryPath.empty())
+        if (m_enginePythonEntryPath.empty() ||
+            m_toolScriptPath.empty() ||
+            m_toolQtBinDirectoryPath.empty() ||
+            m_toolConfigDirectoryPath.empty() ||
+            m_toolLogDirectoryPath.empty())
         {
+            AZ_Error(AWSCoreResourceMappingToolActionName, false,
+                "Failed to get tool launch command, engine python path: %s, tool script path: %s, tool qt binaries path: %s, tool config path: %s, tool log path: %s",
+                m_enginePythonEntryPath.c_str(), m_toolScriptPath.c_str(), m_toolQtBinDirectoryPath.c_str(), m_toolConfigDirectoryPath.c_str(), m_toolLogDirectoryPath.c_str());
             return "";
         }
 
         AZStd::string profileName = "default";
         AWSCoreInternalRequestBus::BroadcastResult(profileName, &AWSCoreInternalRequests::GetProfileName);
 
-        AZStd::string configPath = "";
-        AWSCoreInternalRequestBus::BroadcastResult(configPath, &AWSCoreInternalRequests::GetResourceMappingConfigFolderPath);
-
         if (m_isDebug)
         {
             return AZStd::string::format(
-                "%s debug %s --binaries_path %s --debug --profile %s --config_path %s", m_enginePythonEntryPath.c_str(),
-                m_toolScriptPath.c_str(), m_toolQtBinDirectoryPath.c_str(), profileName.c_str(), configPath.c_str());
+                "%s debug -B %s --binaries_path %s --debug --profile %s --config_path %s --log_path %s",
+                m_enginePythonEntryPath.c_str(), m_toolScriptPath.c_str(), m_toolQtBinDirectoryPath.c_str(),
+                profileName.c_str(), m_toolConfigDirectoryPath.c_str(), m_toolLogDirectoryPath.c_str());
         }
         else
         {
             return AZStd::string::format(
-                "%s %s --binaries_path %s --profile %s --config_path %s", m_enginePythonEntryPath.c_str(),
-                m_toolScriptPath.c_str(), m_toolQtBinDirectoryPath.c_str(), profileName.c_str(), configPath.c_str());
+                "%s -B %s --binaries_path %s --profile %s --config_path %s --log_path %s",
+                m_enginePythonEntryPath.c_str(), m_toolScriptPath.c_str(), m_toolQtBinDirectoryPath.c_str(),
+                profileName.c_str(), m_toolConfigDirectoryPath.c_str(), m_toolLogDirectoryPath.c_str());
```

Comment:
These paths need to be placed in double quotes to deal with paths that contain spaces in them